SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/PartitionManager.cpp

```diff
@@ -1,13 +1,9 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Scary looking diff, mostly the result of creating the PartitionManagerImpl.h file, and all the whitespace edition it involved.